app/_includes/footer.html

```diff
@@ -1,3 +1,7 @@
+{% if page.hide_subscribe == null %}
```

Comment:
Note, every page displays this form. The only page with `hide_subscribe: true` is the defunct enterprise page.